web/history.md

```diff
@@ -2,7 +2,9 @@
 
 ## 13.0 alpha
 * Start version 13.0
-
+* Bug Fix: Default wordbreaker was not working correctly (#2405)
+* Bug Fix: Firefox on macOS was not being detected correctly (#2379)
```

Comment:
These belong in the "stable" section... though I guess there is the question of whether or not "stable" is still paused.